CONTRIBUTING.md

```diff
@@ -377,6 +377,11 @@ Now code linters and formatters will be run each time you commit changes.
 
 You can skip these checks with `git commit --no-verify` or with the short version `git commit -n`.
 
+cuDF also uses [codespell](https://github.com/codespell-project/codespell) to find spelling
+mistakes, and this check is run as part of the pre-commit hook. To apply the suggested spelling
+fixes, you can run  `codespell -i 3 -w .` from the commandline in the cuDF root directory.
```

Comment:
```suggestion
fixes, you can run  `codespell -i 3 -w .` from the command-line in the cuDF root directory.
```
I've not seen command-line as one word before.
https://en.wikipedia.org/wiki/Command-line_interface